cli/samples/voting-demo/README.md

```diff
@@ -1,3 +1,3 @@
 # voting-demo
 
-This sample is the i2kit equivalent of the well known [docker voting sample application](https://github.com/tutumcloud/voting-demo/blob/master/tutum.yml).
+This sample is the i2kit equivalent of the well known [docker voting sample application](https://github.com/dockersamples/example-voting-app).
```

Comment:
I would add a comment explaining that we are using pods for `results.yml` to illustrate this use case, but it could be defined as different services.